src/validateUrl.js

```diff
@@ -0,0 +1,33 @@
+const { errorMessages } = require('./errorMessages');
+
+const caseNames = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const validateUrl = (text, toCase) => {
+  const errors = [];
+
+  if (!text) {
+    errors.push(
+      {
+        message: errorMessages.MissingText,
+      },
+    );
+  }
+
+  if (!toCase) {
+    errors.push(
+      {
+        message: errorMessages.MissingCase,
+      },
+    );
+  } else if (!caseNames.includes(toCase)) {
```

Comment:
This if statement we can separate too. Like, we have 3 different if statements for different situations.